sympy/calculus/util.py

```diff
@@ -49,13 +49,15 @@ def continuous_domain(f, symbol, domain):
 
     Returns
     =======
-    Intervals where the function is continuous.
+
+    interval
```

Comment:
Is this a sympy type?

Reply:
Sorry, my bad. So, now I've looked at [test_util.py](https://github.com/sympy/sympy/blob/master/sympy/calculus/tests/test_util.py) and it seems that the correct types for both the argument "domain" and the return type are `Interval` (I also looked at `sets/sets.py`). I have made another commit. Hope it's correct